lenskit/algorithms/als.py

```diff
@@ -502,16 +494,23 @@ def _train_iters(self, current, uctx, ictx):
 
     def predict_for_user(self, user, items, ratings=None):
         scores = None
+        u_offset = None
         if ratings is not None and len(ratings) > 0:
-            u_offset = None
             if self.bias:
                 ratings, u_offset = self.bias.transform_user(ratings)
 
             ri_idxes = self.item_index_.get_indexer_for(ratings.index)
             ri_good = ri_idxes >= 0
             ri_it = ri_idxes[ri_good]
             ri_val = ratings.values[ri_good]
-            u_feat = _train_bias_row_lu(ri_it, ri_val, self.item_features_, self.regularization)
+
+            # unpack regularization
+            if isinstance(self.regularization, tuple):
+                ureg, ireg = self.regularization
+            else:
+                ureg = ireg = self.regularization
+
+            u_feat = _train_bias_row_lu(ri_it, ri_val, self.item_features_, ureg)
```

Comment:
Why is this getting regularization, instead of `self.OtOr`?

Reply:
Wait, this is for the other one. This fixes an outstanding issue from previous things?

Reply:
Yes, I found that if we sent a tuple in the regularization, then BiasedMF would fail.